rmgpy/rmg/model.py

```diff
@@ -1171,11 +1171,12 @@ def thermoFilterSpecies(self, spcs):
         """
         Tmax = self.Tmax
         for spc in spcs:
-            G = spc.thermo.getFreeEnergy(Tmax)
-            if G > self.Gfmax:
-                Gn = (G-self.Gmax)/(self.Gmax-self.Gmin)
-                logging.info('Removing species {0} with Gibbs energy {1} from edge because it\'s Gibbs number {2} is greater than the toleranceThermoKeepSpeciesInEdge of {3} '.format(spc,G,Gn,self.toleranceThermoKeepSpeciesInEdge))
-                self.removeSpeciesFromEdge(self.reactionSystems,spc)
+            if spc.thermo:
```

Comment:
As we discussed briefly, I don't think this is an appropriate fix, since this means that we will simply never do thermo pruning because thermo is generated later. Instead, it might make sense to move when we do the thermo pruning.